packages/legacy-workbench/src/css/bootstrap-graphdb-theme.css

```diff
@@ -749,10 +759,17 @@ code {
 /*-------------------------------------------------------------
 	Popovers
 -------------------------------------------------------------*/
+.popover,
+.popover pre {
+    color: var(--gw-datatable-row-color);
+    background-color: var(--gw-datatable-row-background);
+}
+
 .popover {
     border-radius: 0;
-    box-shadow: 1px 3px 6px rgba(0, 0, 0, 0.2);
-    border: 1px solid rgba(0, 0, 0, .125);
+    -webkit-box-shadow: var(--gw-dialog-shadow-1-offset-x) var(--gw-dialog-shadow-1-offset-y) var(--gw-dialog-shadow-1-blur) var(--gw-dialog-shadow-1-spread) var(--gw-dialog-shadow-1-color);
+    box-shadow: var(--gw-dialog-shadow-1-offset-x) var(--gw-dialog-shadow-1-offset-y) var(--gw-dialog-shadow-1-blur) var(--gw-dialog-shadow-1-spread) var(--gw-dialog-shadow-1-color);
+    border: var(--gw-datatable-header-border-width) solid var(--gw-datatable-border-color);
```

Comment:
here again you've used datatable related variables in the popover

Reply:
Fixed. The colors are now linked to the dialog variables.